<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="4" passed="4">
    <reporter-output>
    </reporter-output>
    <suite name="Command line suite" duration-ms="79" started-at="2016-09-09T18:08:40Z"
           finished-at="2016-09-09T18:08:40Z">
        <groups>
        </groups>
        <test name="Command line test" duration-ms="79" started-at="2016-09-09T18:08:40Z"
              finished-at="2016-09-09T18:08:40Z">
            <class name="ScalaTest.ShouldBeTest">
                <test-method status="PASS" signature="initialize()[pri:0, instance:ScalaTest.ShouldBeTest@69aa5cb4]"
                             name="initialize" is-config="true" duration-ms="12" started-at="2016-09-09T11:08:40Z"
                             finished-at="2016-09-09T11:08:40Z">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- initialize -->
                <test-method status="PASS" signature="verifyEasy()[pri:0, instance:ScalaTest.ShouldBeTest@69aa5cb4]"
                             name="verifyEasy" duration-ms="11" started-at="2016-09-09T11:08:40Z"
                             finished-at="2016-09-09T11:08:40Z">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- verifyEasy -->
                <test-method status="PASS" signature="initialize()[pri:0, instance:ScalaTest.ShouldBeTest@69aa5cb4]"
                             name="initialize" is-config="true" duration-ms="3" started-at="2016-09-09T11:08:40Z"
                             finished-at="2016-09-09T11:08:40Z">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- initialize -->
                <test-method status="PASS"
                             signature="verifyException()[pri:0, instance:ScalaTest.ShouldBeTest@69aa5cb4]"
                             name="verifyException" duration-ms="8" started-at="2016-09-09T11:08:40Z"
                             finished-at="2016-09-09T11:08:40Z">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- verifyException -->
                <test-method status="PASS" signature="initialize()[pri:0, instance:ScalaTest.ShouldBeTest@69aa5cb4]"
                             name="initialize" is-config="true" duration-ms="0" started-at="2016-09-09T11:08:40Z"
                             finished-at="2016-09-09T11:08:40Z">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- initialize -->
                <test-method status="PASS" signature="verifyFun()[pri:0, instance:ScalaTest.ShouldBeTest@69aa5cb4]"
                             name="verifyFun" duration-ms="10" started-at="2016-09-09T11:08:40Z"
                             finished-at="2016-09-09T11:08:40Z">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- verifyFun -->
                <test-method status="PASS" signature="initialize()[pri:0, instance:ScalaTest.ShouldBeTest@69aa5cb4]"
                             name="initialize" is-config="true" duration-ms="0" started-at="2016-09-09T11:08:40Z"
                             finished-at="2016-09-09T11:08:40Z">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- initialize -->
                <test-method status="PASS" signature="verifyShould()[pri:0, instance:ScalaTest.ShouldBeTest@69aa5cb4]"
                             name="verifyShould" duration-ms="4" started-at="2016-09-09T11:08:40Z"
                             finished-at="2016-09-09T11:08:40Z">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- verifyShould -->
            </class> <!-- ScalaTest.ShouldBeTest -->
        </test> <!-- Command line test -->
    </suite> <!-- Command line suite -->
</testng-results>
